package.json

```diff
@@ -179,6 +179,15 @@
 						}
 					]
 				},
+				"githubPullRequests.pullRequestLabels": {
```

Comment:
I think this setting needs to include the word "create". Maybe `githubPullRequests.labelCreated` for consistency with 

https://github.com/umakantv/vscode-pull-request-github/blob/19f1d00724fbb7d78527976548922fc930cc1bdb/package.json#L308-L311

Reply:
Done